lib/trento_web/openapi/v2/schema/cluster.ex

```diff
@@ -99,6 +100,11 @@ defmodule TrentoWeb.OpenApi.V2.Schema.Cluster do
             description: "HANA architecture type.",
             enum: HanaArchitectureType.values()
           },
+          hana_scenario: %Schema{
+            type: :string,
+            description: "Detected hana scenario",
```

Comment:
I would simply put `HANA scenario type`.
PD: `HANA` always goes in capital letters